config.example.wikimedia.yaml

```diff
@@ -39,7 +39,7 @@ wikimedia.org: &wikimedia.org
     - path: projects/wikimedia.org.yaml
       options:
         <<: *default_options
-        pageviews:
+        metrics:
```

Comment:
Yes, but the change is happening in `/projects/wikimedia.org` which is located in this repo, so all is needed here is a deploy of RESTBase. To go the extra step of renaming `options.pageviews` to `options.metrics` we'd need a coordinated config and repo deploy of the front-end RB, which involves a patch to `ops/puppet` and coordination with Ops.